src/xcode/ENA/ENA/Source/Scenes/Events/TraceLocations/Configuration/TraceLocationConfigurationViewController.swift

```diff
@@ -277,6 +277,12 @@ class TraceLocationConfigurationViewController: UIViewController, FooterViewHand
 
 	@IBAction private func temporaryDefaultLengthSwitchToggled(_ sender: UISwitch) {
 		viewModel.temporaryDefaultLengthSwitchSet(to: sender.isOn)
+		
+		if sender.isOn == true && UIScreen.main.bounds.size.height <= 667 {
```

Comment:
Will this work with different screen sizes? I don't think so

Reply:
Yes, It works! I have tested it in iPhone SE 2nd gen (375*667) & bigger screen like on iPhone Xs (375*812).